accepted/2022/FunctionPointers.md

```diff
@@ -0,0 +1,431 @@
+# Function Pointers
+#### Steve Harter
+#### December 13, 2022
+
+# Background
+For additional context, see:
+- The [original feature ask](https://github.com/dotnet/runtime/issues/11354)
+- The [MetadataLoadContext feature ask](https://github.com/dotnet/runtime/issues/43791)
+- The [7.0 introspection attempt](https://github.com/dotnet/runtime/pull/71516)
+- [C# function pointer spec](https://learn.microsoft.com/dotnet/csharp/language-reference/proposals/csharp-9.0/function-pointers)
+
+Function pointers, which have been in ECMA-335 and used by the CLR since the beginning were only recently supported by C# in C# v9.0 ad .NET v6. However, the .NET type system has not been updated to directly support them and they are currently exposed as the `IntPtr` type:
```

Comment:
```suggestion
Function pointers, which have been in ECMA-335 and used by the CLR since the beginning were only recently supported by C# in C# v9.0 and .NET v6. However, the .NET type system has not been updated to directly support them and they are currently exposed as the `IntPtr` type:
```